themes/default/content/blog/keep-your-secrets-secure-by-default/index.md

```diff
@@ -7,7 +7,6 @@ meta_image: secure_by_default.png
 authors:
     - alex-mullans
 tags:
-    - secrets
```

Comment:
I'll admit that I recently used this tag to find all the posts we had on secrets - and there are quite a few. I'm not sure exactly how it fits with the principles - but I feel like this makes sense as a label (similar to policy-as-code).

Reply:
I can see the case for secrets as a feature. Will bring it back.